interpreter/core/respond.py

```diff
@@ -113,9 +114,12 @@ def respond(interpreter):
 
                 # Get a code interpreter to run it
                 language = interpreter.messages[-1]["language"]
-                if language not in interpreter._code_interpreters:
-                    interpreter._code_interpreters[language] = create_code_interpreter(language)
-                code_interpreter = interpreter._code_interpreters[language]
+                if language in language_map:
+                    if language not in interpreter._code_interpreters:
+                        interpreter._code_interpreters[language] = create_code_interpreter(language)
+                    code_interpreter = interpreter._code_interpreters[language]
+                else:
+                    print(f"{language} not in supported languages.")
```

Comment:
What should happen when language is not in language_map?

Reply:
Probably something like what you have here where we say something like: `Error: Open Interpreter does not currently support {language}.` and then we might need to yield that as output to make sure open interpreter recognizes the issue.